chainercv/experimental/links/model/fcis/fcis_resnet101.py

```diff
@@ -0,0 +1,234 @@
+            loc_normalize_std=loc_normalize_std,
+            iter2=iter2)
+
+        mean = np.array([123.15, 115.90, 103.06],
+                        dtype=np.float32)[:, None, None]
+
+        super(FCISResNet101, self).__init__(
+            extractor, rpn, head,
+            mean, min_size, max_size,
+            loc_normalize_mean, loc_normalize_std)
+
+        if pretrained_model in self._models:
+            path = download_model(self._models[pretrained_model]['url'])
+            chainer.serializers.load_npz(path, self)
+
+
+class ResNet101Extractor(chainer.Chain):
+
+    def __init__(self):
+        super(ResNet101Extractor, self).__init__()
+        initialW = chainer.initializers.HeNormal()
+        kwargs = {
+            'initialW': initialW,
+            'bn_kwargs': {'eps': 1e-5},
+            'stride_first': True
+        }
+
+        with self.init_scope():
+            # ResNet
+            self.conv1 = Conv2DBNActiv(
+                3, 64, 7, 2, 3, nobias=True, initialW=initialW)
+            self.pool1 = lambda x: F.max_pooling_2d(x, ksize=3, stride=2)
+            self.res2 = ResBlock(3, 64, 64, 256, 1, **kwargs)
+            self.res3 = ResBlock(4, 256, 128, 512, 2, **kwargs)
+            self.res4 = ResBlock(23, 512, 256, 1024, 2, **kwargs)
+            self.res5 = ResBlock(3, 1024, 512, 2048, 1, 2, **kwargs)
+
+    def __call__(self, x):
+        with chainer.using_config('train', False):
+            with chainer.function.no_backprop_mode():
+                h = self.pool1(self.conv1(x))
+                h = self.res2(h)
+            h = self.res3(h)
+            res4 = self.res4(h)
+            res5 = self.res5(res4)
+        return res4, res5
+
+
+class FCISResNet101Head(chainer.Chain):
+
+    def __init__(
+            self,
+            n_class,
+            roi_size, group_size, spatial_scale,
+            loc_normalize_mean, loc_normalize_std,
+            iter2
+    ):
+        super(FCISResNet101Head, self).__init__()
+
+        self.n_class = n_class
+        self.spatial_scale = spatial_scale
+        self.group_size = group_size
+        self.roi_size = roi_size
+        self.loc_normalize_mean = loc_normalize_mean
+        self.loc_normalize_std = loc_normalize_std
+        self.iter2 = iter2
+
+        initialW = chainer.initializers.Normal(0.01)
+        with self.init_scope():
+            self.psroi_conv1 = L.Convolution2D(
+                2048, 1024, 1, 1, 0, initialW=initialW)
+            self.psroi_conv2 = L.Convolution2D(
+                1024, group_size * group_size * n_class * 2,
+                1, 1, 0, initialW=initialW)
+            self.psroi_conv3 = L.Convolution2D(
+                1024, group_size * group_size * 2 * 4,
+                1, 1, 0, initialW=initialW)
+
+    def __call__(self, x, rois, roi_indices, img_size):
+        h = F.relu(self.psroi_conv1(x))
+        h_cls_seg = self.psroi_conv2(h)
+        h_ag_locs = self.psroi_conv3(h)
```

Comment:
How about `h_ag_loc` instead of `h_ag_loc`.
We usually do not put `s` with variables starting with `h_` (i.e., not `hs`).
Also, this looks more consistent with `h_cls_seg`, which you did not put `s`.

Reply:
OK